src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -36,13 +36,15 @@ def _check_errors(
     :return: Values in model
     """
     if not values.get("errors"):
-        if not all(values.get(required_field) for required_field in required_fields):
+        if not all(
+            values.get(required_field) is not None for required_field in required_fields
+        ):
             err_msg = f"{required_fields} must all be provided"
             raise ValueError(err_msg)
 
         if either_or_fields:
             for field1, field2 in either_or_fields:
-                if not (values.get(field1) or values.get(field2)):
+                if not (values.get(field1) is not None or values.get(field2)):
```

Comment:
@korikuzma could you add parentheses here to ensure the intended logical order is clear? ie 

```python
                if not ((values.get(field1) is not None) or values.get(field2)):
```

assuming I'm reading this right

Reply:
I messed this one up. I should be checking both